collectiondata.sql

```diff
@@ -0,0 +1,55 @@
+# ************************************************************
+# Sequel Pro SQL dump
+# Version 4541
+#
+# http://www.sequelpro.com/
+# https://github.com/sequelpro/sequelpro
+#
+# Host: 127.0.0.1 (MySQL 5.7.29)
+# Database: collection
+# Generation Time: 2020-03-23 14:11:36 +0000
+# ************************************************************
+
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!40101 SET NAMES utf8 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+
+# Dump of table characterinfo
+# ------------------------------------------------------------
+
+DROP TABLE IF EXISTS `characterinfo`;
+
+CREATE TABLE `characterinfo` (
+  `id` int(11) unsigned NOT NULL AUTO_INCREMENT,
+  `name` varchar(255) DEFAULT NULL,
+  `blood-type` varchar(255) DEFAULT NULL,
+  `house` varchar(255) DEFAULT NULL,
```

Comment:
as the house can only be 1 of 4 values, i would create a seperate table with the 4 values in, then make this field a id link to that table. otherwise you are storing a lot of data and there is no data validation